templates/project.py

```diff
@@ -439,6 +450,10 @@ def generate_config(context):
 
   project_id = context.properties.get('projectId')
   project_name = context.properties.get('name', project_id)
+  project_labels = context.properties.get('labels', {})
+  project_labels.update({
+      "billingaccount": label_safe_string(context.properties.get('billingAccountFriendlyName'))
```

Comment:
naming nitpick: s/billingaccount/billing-account/? It's so much easier to remember keys when words are separated.

Maaaaybe also add a "name" suffix to make clear this isn't the billing account ID.

Reply:
I was mirroring what we do in FC today, which is call it `billingaccount`. I'm not sure if it's there for funsies or if it's looked up by something.